editors/code/src/lsp.ts

```diff
@@ -30,11 +34,17 @@ export class Lsp {
 
 	private fileSettings: FileSettingsState;
 
+	public onSettingsNotificationEmitter: vscode.EventEmitter<void>;
```

Comment:
Place the `Emitter` near the event itself?

Reply:
hmm this should actually be `private` and I think we keep private fields at the bottom?